emote/memory/memory.py

```diff
@@ -463,6 +464,58 @@ def __iter__(self):
             yield {self.data_group: data, self.size_key: data[self.size_key]}
 
 
+class JointMemoryLoader:
+    """A memory loader capable of loading data from multiple `MemoryLoader`s."""
+
+    def __init__(self, loaders: list[MemoryLoader], size_key: str = "batch_size"):
+        self._loaders = loaders
+        self._size_key = size_key
+
+        datagroups = (loader.data_group for loader in loaders)
```

Comment:
This seems quite ineffectual, can't you just put it in the Counter call?

Reply:
Indeed!